code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -35,3 +35,12 @@
 			adjust_nutrition(-(HUNGER_FACTOR/10))
 			if(m_intent == MOVE_INTENT_RUN)
 				adjust_nutrition(-(HUNGER_FACTOR/10))
+	
+/mob/living/carbon/carefulmovement()
```

Comment:
"careful movement" sounds more like an action than a check. Should probably be renamed to make it obvious it's checking if the user is moving carefully